lit/Reproducer/Swift/TestSwiftInterface.test

```diff
@@ -0,0 +1,53 @@
+# Test that reproducers can deal with .swiftinterface files.
+
+# REQUIRES: system-darwin
+
+# Start clean.
+# RUN: rm -rf %t.build && mkdir %t.build && cd %t.build
+# RUN: rm -rf %t.lib && mkdir %t.lib
+# RUN: cp %S/Inputs/A.swift %t.build/AA.swift
+# RUN: cp %S/Inputs/Main.swift %t.build/Main.swift
+
+# RUN: %target-swiftc -module-name AA \
+# RUN:          -module-cache-path %t.build/cache \
+# RUN:          -enable-library-evolution \
+# RUN:          -emit-parseable-module-interface-path %t.lib/AA.swiftinterface \
```

Comment:
Nit: Can you use `-emit-module-interface-path`? It's the final name of this flag.